Habitica/res/menu/menu_full_profile.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    tools:context=".ui.activities.TaskFormActivity">
+    <item android:id="@+id/private_message"
+        android:title="PM"
```

Comment:
This title should be spelled out, since users relying on accessibilty options might need it